packages/hash/api/src/index.ts

```diff
@@ -132,13 +137,20 @@ const main = async () => {
     port: graphApiPort,
   });
 
+  await ensureSystemOrgAccountIdExists({ graphApi, logger });
+
   await ensureSystemTypesExist({ graphApi, logger });
 
+  await ensureSystemOrgExists({
+    graphApi,
+    logger,
+  });
+
   await ensureSystemEntitiesExists({ graphApi, logger });
 
   // This will seed users, an org and pages.
   // Configurable through environment variables.
-  await seedOrgsAndUsers({ graphApi, logger });
+  await seedUsers({ graphApi, logger, orgModel: systemOrgModel });
```

Comment:
I understand why `systemOrgModel` is exported as a mutable export, but it does feel strange/coupled that the `ensureSystemOrgAccountIdExists` and `ensureSystemOrgExists` functions don't return the accountId/orgModel to be used here instead of importing

Reply:
Agreed, see my comment to Alfie on why I stuck with this approach in https://github.com/hashintel/hash/pull/1596#discussion_r1041001326